packages/gravity-ui-nunjucks/package.json

```diff
@@ -47,7 +47,7 @@
   "devDependencies": {
     "@buildit/eslint-config": "^1.0.0",
     "@buildit/gravity-particles": "^0.3.0",
-    "@buildit/gravity-ui-web": "^3.0.0-alpha.2",
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.6",
```

Comment:
Is there not an equivalent `package-lock.json` change for the nunjucks package?

Reply:
Nope.

When Lerna does it's "bootstrap" thing, it temporarily removes any local packages dependencies, runs the install and then restores the `package.json`. So, the `package-lock.json` files don't contain refrences to any of the other packages in this monorepo

Reply:
I suspect I didn't really need to update the Nunjucks `package.json` at all, but it felt neater to keep it up-to-date.

Especially later, when we start publishing that to NPM, it'll be important for it's `package.json` to list the correct version of `gravity-ui-web`

Reply:
Makes sense, i saw the `gravity-ui-web` lock being updated but of course that's the actual package's version.